src/main/java/com/lorepo/icplayer/client/module/text/TextView.java

```diff
@@ -217,6 +209,22 @@ private native boolean isElementContainedInBody(Element e)/*-{
 		return $doc.body.contains(e);
 	}-*/;
 
+	private void setGapWidth(GapWidget gap, int width) {
```

Comment:
Proponuję ujednolicić jakie argumenty przyjmuje ta metoda i jej nazwę, względem tej po stronie editora.
Po stronie playera to lepiej aby ta metoda się nazywała setGapMinWidth
Co do argumentów to podkreślę, że warto, jednak nie jest to konieczne.